cross/libthreadar/Makefile

```diff
@@ -8,7 +8,7 @@ PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 DEPENDS =
 
 # archs without C++11 support are not supported:
-UNSUPPORTED_ARCHS = $(ARM5_ARCHES) hi3535 powerpc ppc824x ppc853x ppc854x
+UNSUPPORTED_ARCHS = $(ARMv5_ARCHS) $(ARMv7L_ARCHS) $(OLD_PPC_ARCHS)
```

Comment:
I've been seing ARMv7L_ARCHS often included with ARMv5 + PPC_LEGACY... could ARMv7L be simply added to the generic LEGACY_ARCHS based on my proposed commit?